libethcore/KeyManager.cpp

```diff
@@ -342,6 +342,19 @@ string const& KeyManager::accountName(Address const& _address) const
 	}
 }
 
+void KeyManager::changeName(Address const& _address, std::string const& _name)
+{
+	try
+	{
+		m_keyInfo.at(_address).accountName = _name;
+	}
+	catch (...)
+	{
+		return;
+	}
+	write(m_keysFile);
```

Comment:
Sorry, but I am not convinced that this function can not throw. 
In `KeyManger::write` the function `writeFile` is called, where is this function defined?